packages/front-end/components/Settings/NewDataSourceForm.tsx

```diff
@@ -565,6 +616,34 @@ const NewDataSourceForm: FC<{
                   </div>
                 </div>
               )}
+              {showInbuiltDatasource && (
+                <div
+                  className={clsx(
+                    styles.ctaContainer,
+                    showImportSampleData && showInbuiltDatasource
+                      ? "w-25"
+                      : showImportSampleData || showInbuiltDatasource
+                      ? "w-33"
+                      : "w-50"
+                  )}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    callCreateInbuiltDatasource();
+                  }}
+                >
+                  {isCreatingInbuiltDatasource && <LoadingOverlay />}
+                  <div className={styles.ctaButton}>
+                    <h3 className={styles.ctaText}>
+                      Use Growthbook&apos;s Warehouse
+                    </h3>
+                    <p>
+                      If you don&apos;t have your own datasource you can use
+                      Growthbook&apos;s own warehouse to house your event
+                      tracking data
+                    </p>
+                  </div>
+                </div>
+              )}
```

Comment:
Thinking about this more, I would revert this entire file.  The final UI will be completely different anyway.  

If we need a quick button for us to set it up on our Cloud org, we can add something to the `admin.tsx` page or even just make the POST call manually.